EIPS/eip-royalty-treasury-proposal.md

```diff
@@ -0,0 +1,73 @@
+---
+eip: royalty-treasury-proposal
```

Comment:
```suggestion
eip: 6145
```
Don't forget to rename the file!

Reply:
@Pandapip1 should this be #6144 as this was the original PR?

Reply:
No that md was corrupt upon commit, thus rejected. Please continue with 6145